diode-proto/diode/v1/reconciler.proto

```diff
@@ -77,22 +86,21 @@ message IngestionLog {
 
 // The request to retrieve ingestion logs
 message RetrieveIngestionLogsRequest {
-  int32 page_size = 1 [(validate.rules).int32 = {
-    gte: 1
-    lte: 1000
-  }]; // Number of logs per page, default is 100
+  optional int32 page_size = 1; // Number of logs per page, default is 100
   optional State state = 2; // Optional filter by state field
   string data_type = 3; // Optional filter by data type field
   string request_id = 4; // Optional filter by request ID
   int64 ingestion_ts_start = 5; // Optional start of ingestion timestamp range
   int64 ingestion_ts_end = 6; // Optional end of ingestion timestamp range
   string page_token = 7; // Token to fetch the next page of results
+  bool summary = 8; // Optional flag to return all stats
```

Comment:
s/summary/include_metrics ?

Reply:
hmm, the way I've implemented it was that if it is passed we only return the summary and return, no log returned.. 'include_metrics' seems to give the impression that we should return specified logs and also 'summary'

Reply:
Got it, thanks for clarification, missing `only` in the comment tripped me over.  `metrics_only` sounds better?

Reply:
changed it!